app/src/App.tsx

```diff
@@ -14,9 +17,13 @@ const App = () => {
     <FullHeight>
       <StoreProvider store={store}>
         <ThemeProvider>
-          <Routes />
-          <AlertContainer />
-          <TourHost />
+          <Suspense fallback={<Loading delay={500} />}>
```

Comment:
Refactored `AppRoutes` to just be routes. Moved the `Suspense` and `Router` a level up.
This also helped fix issue with `AppRoutes.spec.tsx` as we can no longer have nested `Router`s (`renderWithProviders` was wrapping `AppRouter` in another `Router`).